contracts/relayer/RelayedAragonAppWithParameterizedSender.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.24;
+
+import "../apps/AragonApp.sol";
+
+
+contract RelayedAragonAppWithParameterizedSender is AragonApp {
```

Comment:
Is this supposed to be the 4th case in the description? (The "Parameterized App Relayer") Isn't it missing the `exec` bits?

Reply:
No, this is a relayed app, i.e. an app that is being relayed by an on-chain relayer.